#request
int8 START_FOLLOWING = 1
int8 STOP_FOLLOWING = 0
int8 LOCK = 2
int8 UNLOCK = 3
int8 state
---
#response


